docs/notebooks/index.rst

```diff
@@ -0,0 +1,12 @@
+Examples
+========
+
+.. toctree::
+    :maxdepth: 1
```

Comment:
```suggestion
    :maxdepth: 1
    :numbered:
```
Suggestion, so it makes the documentation structure even clearer, with numbered sections.